09-features.md

```diff
@@ -31,9 +31,10 @@ These flags may only be used in the `init` message:
 
 The following `globalfeatures` bits are currently assigned by this specification:
 
-| Bits | Name              | Description                                                        | Link                                  |
-|------|-------------------|--------------------------------------------------------------------|---------------------------------------|
-| 8/9  | `var_onion_optin` | This node requires/supports variable-length routing onion payloads | [Routing Onion Specification][bolt04] |
+| Bits  | Name              | Description                                                        | Link                                  |
+|-------|-------------------|--------------------------------------------------------------------|---------------------------------------|
+| 8/9   | `var_onion_optin` | This node requires/supports variable-length routing onion payloads | [Routing Onion Specification][bolt04] |
+| 12/13 | `option_amp`      | Payee supports AMP                                                 | [BOLT #4](04-onion-routing#atomic-multi-path-payments)
```

Comment:
I wonder if we need a way to signal or communicate interdependent feature bits, as the `option_amp` bits are dependent on the node also supporting the `var_onion_optin` bit as well. On the other-hand, one could say that `option_amp` implies the other, though some clients may be stricter and only send an AMP payment if the var onion bit is set as well. 

This also needs to define new feature bits and possibly tags in the BOLT 11 invoices as well.

Reply:
resolved in #719